ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewGroup.java

```diff
@@ -366,7 +380,13 @@ private void updateSubviewClipStatus(Rect clippingRect, int idx, int clippedSoFa
     // it won't be size and located properly.
     Animation animation = child.getAnimation();
     boolean isAnimating = animation != null && !animation.hasEnded();
-    if (!intersects && child.getParent() != null && !isAnimating) {
+
+    boolean preventClipping = false;
+    if (child instanceof ReactViewGroup) {
+      preventClipping = ((ReactViewGroup)child).getPreventClipping();
+    }
+
+    if (!intersects && child.getParent() != null && !isAnimating && !preventClipping) {
```

Comment:
Did you confirm that scrolling down for a bit and then scrolling back up doesn't break things?  In our testing it wasn't sufficient to only insert things here.